include/ZemachFormalism.h

```diff
@@ -91,7 +91,11 @@ class ZemachFormalism : public SpinAmplitudeCache
     /// \param L orbital angular momentum
     /// \param two_S 2 * the total spin angular momentum
     virtual std::shared_ptr<SpinAmplitude> create(unsigned two_J, const SpinVector& two_j, unsigned l, unsigned two_s) const override
-    { return (two_J == 0) ? unit(two_J, two_j, l, two_s) : std::shared_ptr<SpinAmplitude>(new ZemachSpinAmplitude(two_J, two_j, l, two_s)); }
+    {
```

Comment:
`ZemachSpinAmplitude` still needs to be quasi-two-body. So we need to remove the `two_j.size() < 3` check.

We should add to `SpinAmplitudeCache::spinAmplitude(...)` that it checks the size of `two_j` and calls `unit(...)` to create the proper `UnitSpinAmplitude` for a non-resonant decay.

`ZemachSpinAmplitude` can handle the situation when it is constant, just as `HelicitySpinAmplitude` does now. It should be constant when `S == 0` and when a `ParticleCombination`'s `indices().size < 3` --- this should be implemented in `amplitude(...)`, which should be overrided. `calculate(...)` should also be overrided to do nothing when `S == 0`. 

Also: I think we can override `DataAccessor::addParticleCombination(...)` in `ZemachSpinAmplitude` to not add `ParticleCombination`'s with fewer than 3 indices.